lib/_http_outgoing.js

```diff
@@ -78,12 +78,14 @@ exports.OutgoingMessage = OutgoingMessage;
 OutgoingMessage.prototype.setTimeout = function(msecs, callback) {
   if (callback)
     this.on('timeout', callback);
+
   if (!this.socket) {
     this.once('socket', function(socket) {
       socket.setTimeout(msecs);
     });
-  } else
+  } else {
     this.socket.setTimeout(msecs);
+  }
```

Comment:
Please undo them, it complicates code archeology.  The guideline is to refrain from stylistic changes unless you're updating the code that you're touching anyway.